### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mfd2007**
All dependencies have been updated.
Some tests have been changed, because of deprecated functions. 

Test pipeline in gh actions seems a bit buggy. Locally there are no errors.

### CODE REVIEW

File: docs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/index.adoc
+/index.html
```
Comment (tschmidtb51) [inline docs/.gitignore:2]: Would it be useful to have the documentation as part of the repo? Or should it be regenerated in each build? I guess that latter would be better, correct?

**Review by tschmidtb51 — COMMENTED** [review]
Generally, approve. Some detail questions.

File: src/test/java/de/bsi/secvisogram/csaf_cms_backend/OpenApiGenerateDocuTest.java [hunk 2]
```diff
@@ -1,32 +0,0 @@
-package de.bsi.secvisogram.csaf_cms_backend;
```
[the remark this answers is not in the document]
Comment (mfd2007) [reply src/test/java/de/bsi/secvisogram/csaf_cms_backend/OpenApiGenerateDocuTest.java:1]: I moved the generation of the documentation in the gradle process.
In my opinion it is a bit curious to generate the docs with a "test". Now it is a gradle task.

File: docs/.gitignore (hunk 1 above)
Comment (mfd2007) [reply docs/.gitignore:2]: No, it would not include it in the repo, because when starting, the docs are also available on the server via swagger.

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Pull request merged**